db/migrate/20200110121522_create_user_api_tokens.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class CreateUserApiTokens < ActiveRecord::Migration[5.2]
+  def change
+    create_table :user_api_tokens do |t|
+      t.references :user
+      t.string :name
+      t.string :token
```

Comment:
should we add a unique key in the database?

Reply:
You are right

Reply:
Fixed!